### PR TITLE
new sln

### PR DESCRIPTION
**Pull request by meaghanlewis**

### CODE REVIEW

File: ConsoleApp2/ConsoleApp2.sln [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+VisualStudioVersion = 15.0.27428.2002
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{F184B08F-C81C-45F6-A57F-5ABD9991F28F}") = "ConsoleApp2", "ConsoleApp2\ConsoleApp2.vbproj", "{8BC81A56-B385-44F7-88EA-DE806F53AD62}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
```
Comment (meaghanlewis) [inline ConsoleApp2/ConsoleApp2.sln:13]: yes

File: ConsoleApp2/ConsoleApp2.sln [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+		{8BC81A56-B385-44F7-88EA-DE806F53AD62}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{8BC81A56-B385-44F7-88EA-DE806F53AD62}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{8BC81A56-B385-44F7-88EA-DE806F53AD62}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{8BC81A56-B385-44F7-88EA-DE806F53AD62}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
```
Comment (meaghanlewis) [inline ConsoleApp2/ConsoleApp2.sln:21]: no

File: ConsoleApp2/ConsoleApp2/ConsoleApp2.vbproj [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```
Comment (meaghanlewis) [inline ConsoleApp2/ConsoleApp2/ConsoleApp2.vbproj:5]: cool

File: ConsoleApp2/ConsoleApp2/Program.vb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Imports System
+
+Module Program
+    Sub Main(args As String())
+        Console.WriteLine("Hello World!")
+    End Sub
+End Module
```
Comment (ghfvs-tester) [inline ConsoleApp2/ConsoleApp2/Program.vb:7]: LG

File: ConsoleApp2/ConsoleApp2/Program.vb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+Imports System
+
+Module Program
```
Comment (ghfvs-tester) [inline ConsoleApp2/ConsoleApp2/Program.vb:3]: ok

**Review by ghfvs-tester — COMMENTED** [review]
thanks for these changes

**Review by ghfvs-tester — COMMENTED** [review]
:(

**Review by meaghanlewis — COMMENTED** [review]
comment review

**Pull request merged**